projects/solar/src/sense_mcp3427.c

```diff
@@ -0,0 +1,84 @@
+#include "sense_mcp3427.h"
+#include "data_store.h"
```

Comment:
nit: I guess the section thing isn't that strictly enforced lol, my b

Reply:
Ah whoops, I think the section thing was decided after this was written and I forgot to update it, my bad.